test/test_mypy.py

```diff
@@ -309,6 +309,9 @@ def test_son_document_type(self) -> None:
         assert retreived is not None
         retreived["a"] = 1
 
+    def test_son_document_type_runtime(self) -> None:
+        client = MongoClient(document_class=SON[str, Any])
```

Comment:
Add connect=False to avoid starting threads.

Reply:
Fixed